src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -63,16 +65,18 @@
     private static double kD = Preferences.getInstance().getDouble(RobotMap.Preferences.K_D, 1);
     private static double kF = Preferences.getInstance().getDouble(RobotMap.Preferences.K_F, 1);
 
+    private Lidar liftLidar = new Lidar(Port.kOnboard, 0x60);
+
     //enumerator that will be pulled from for the GoToHeight Command
-    public enum LiftHeight {
+    public enum DesiredLiftHeight {
```

Comment:
I don't think we want to change this. These are just values so maybe LiftHeights, but I wouldn't call this DesiredLiftHeight.